grpc/diatheke.proto

```diff
@@ -281,10 +303,40 @@ message PushTextRequest {
   string text = 2;
 }
 
+// Request to change the current story of a session.
+message StoryRequest {
+  // ID of the session that will have its story changed.
+  string session_id = 1;
```

Comment:
Is the `session_id` required ? SetStory can only alter whatever the current session's state is right ?

Reply:
Yes, it is required. The server can host multiple sessions. You might think of a session as instantiating a model, with each session having its own memory and state. We want to change only the current state of specific session, and not all instances where a model is being used.

Reply:
Oh, each client would be managing one active session though right ?

Reply:
Not necessarily. A client could have as many sessions as they want. It is not a current restriction that a client is limited to a single session. Whether it _should_ be that way or not is a separate discussion.

Reply:
Ah got it, thanks for clarifying